index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My first Portfolio</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+</head>
+<body>
+    <header class="toolbar">
+        <h3 class="logo"><link rel="stylesheet" href="#">Moise Mulungu</h3>
+        <div class="bars">
+            <i class="fas fa-bars"></i>
+        </div>
+    </header>
+    <section class="main">
+        <h1 class="intro">Hey there.<br>
+            I'm Moise
+        </h1>
+        <h2 class="title">I'm a software developer</h2>
+        <p class="text"> I can help you build a product, feature or<br>
+            website Look through some of my work<br> 
+            experience! if you like what you see<br>
+            and have a project you needed coded,<br>
+            don't hesitate to contact me.
+        </p>
+        <nav class="contacts">
+            <ul>
+                <li><a href="https://github.com/moise-mulungu/portfolio-mobile-version"><i class="fab fa-github"></i></a></li>
```

Comment:
- [ ] The <nav> tag defines a set of navigation links. That's true.  But following the HTML semantics best practices, it would be better to use `nav` only in the header. 
- I would recommend using only `ul` in this case and giving it the class `contacts`.